data-prepper-plugins/opensearch/src/test/java/com/amazon/dataprepper/plugins/sink/opensearch/index/DefaultIndexManagerTests.java

```diff
@@ -0,0 +1,205 @@
+package com.amazon.dataprepper.plugins.sink.opensearch.index;
+
+import com.amazon.dataprepper.plugins.sink.opensearch.OpenSearchSinkConfiguration;
+import org.junit.After;
```

Comment:
We should be using JUnit5 for all new tests. I won't block the PR for this, but please be sure to use JUnit 5/Jupiter going forward.

Reply:
Sounds good. Currently, the Sink Plugin depends on JUnit4, not Junit5. We can make a pull request to specifically upgrade the JUnit version.